test/javascripts/helpers/format-currency-test.js

```diff
@@ -0,0 +1,12 @@
+import { setupTest } from "ember-qunit";
+import { module, test } from "qunit";
+import { formatCurrency } from "discourse/plugins/discourse-subscriptions/discourse/helpers/format-currency";
+
+module("Unit | Helper | format-currency", function (hooks) {
+  setupTest(hooks);
+
+  test("it formats USD correctly", function (assert) {
```

Comment:
Maybe also add a test to make sure it formats something with more decimal places correctly? Seems like `toFixed` rounds, so `338.289` should be `338.29` for example.